Updated Docs - Added Superuser & Automatic Seeding Data (#21)
Health-Informatics-UoN/carrot: website/src/pages/mapper/dev_guide/quickstart.mdx

}
```

### Environment variables setup for Super User Credentials.

Before running the Docker Compose, decide whether to use **Default Superuser Credentials** or **Custom Superuser Credentials** for logging into the Django Admin Panel.

<div id="superuserCredentials">
  - <u>**Default Super User Creation**</u>: By default, **Carrot creates a superuser** with the following credentials:
    <div>
    - **Username**: `admin` 
    - **Password**: `Password123!`
    </div>
  **Note:** If you wish to use these **default credentials**, you can **skip the setup** for **<u>custom super user credentials</u>**.
</div>

- <u>**Custom Superuser Credentials**</u>: If you **wish to create a new super user with custom credentials**, do the following:
  <div>
    1. Locate the `docker-compose.yml` file in the `root` directory of Carrot.
    2. Open the file and find the `api` service.

>     3. Make changes in the `environment` section of the `api` service as follows:

Reviewer: ```suggestion
    3. Add the following variables to the `environment` section of the `api` service:
```

Author: Ok

      ```yaml
      environment:
        - SUPERUSER_DEFAULT_USERNAME : your_username
        - SUPERUSER_DEFAULT_EMAIL : your_email
        - SUPERUSER_DEFAULT_PASSWORD : your_password
      ```

    4. Replace `your_username`, `your_email`, and `your_password` with your desired credentials.
  </div>

<Callout type="warning">
  **Custom superuser credentials must be set before running Docker-Compose**. If not, the **default credentials created by the Carrot will be applied**.
</Callout>

Once you change the environment variables, **your custom super user credentials will be replaced with the default super user credentials**.
  <div>

>     - Now you can use your **superuser credentials** to **log in to the Django Admin panel** (steps are mentioned later in this guide).

Reviewer: I don't think we need this line, cause we are repeating this notice

Author: Ok👍🏻

</Callout>

### First instances creation

Create some first instances in Carrot app by logging to Django Admin panel `http://localhost:8000/admin/`.

> 1. You can **logging the Admin Panel** using the [superuser credentials](#superuserCredentials), either the **default credentials** or the **custom credentials** you configured earlier.

Reviewer 2: ```suggestion
1. You can **log in to the Admin Panel** using the [superuser credentials](#superuserCredentials), either the **default credentials** or the **custom credentials** you configured earlier.
```
